### DISCUSSION TITLE
`docker compose` vs `docker-compose` in guides

### DISCUSSION THREAD

**Discussion by ryanrussell** [seaweedfs/seaweedfs#3698] [Q&A] [answered]
This feels more like a discussion item than an issue at this point.

Newer versions of docker ship with `docker compose` instead of needing to install `docker-compose` ...

Do you feel like it's appropriate to update the readme's to the new command at this point?

Here's a quick reference: [SO on docker compose vs docker-compose](https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose)

I believe most people understand the difference and which command they need; it's purely a question of whether the `copy & paste` version in the readme is tailored for `v1` or `v2`.

Thoughts?

**Comment by chrislusf** [comment, marked as answer]
we should use the newer command in the doc.

**Reply by ryanrussell** [reply]
Cool, made #3699 to tag this one
